website/pages/docs/k8s/connect/ingress-gateways.mdx

```diff
@@ -5,4 +5,174 @@ sidebar_title: Ingress Gateways
 description: Configuring Ingress Gateways on Kubernetes
 ---
 
-# Ingress Gateways on Kubernetes
\ No newline at end of file
+# Ingress Gateways on Kubernetes
+
+-> 1.8.0+: This feature is available in Consul versions 1.8.0 and higher
+
+~> This topic requires familiarity with [Ingress Gateways](https://www.consul.io/docs/connect/ingress-gateway).
+
+This page describes how to enable external access to Connect service mesh services running inside Kubernetes using Consul ingress gateways.
+See [Ingress Gateways](https://www.consul.io/docs/connect/ingress-gateway) for more information on use-cases and how it works.
+
+Adding an ingress gateway is a multi-step process, that at a high level, consists of the following steps:
+
+* Setting the helm chart configuration
+* Deploying the helm chart
+* Configuring the gateway
+* Defining an Intention (optional)
+* Deploying your application to Kubernetes
+* Connecting to your application
+
+## Setting the helm chart configuration
+When deploying the helm chart you must provide helm with a custom yaml file that contains your environment configuration.
+The following yaml snippet is the launching point for a valid configuration that must be supplied when installing using the [official consul-helm chart](https://hub.helm.sh/charts/hashicorp/consul).
+Information on additional options can be found [here](https://www.consul.io/docs/k8s/helm). Configuration options for ingress gateways reside under the [ingressGateways](https://www.consul.io/docs/k8s/helm#v-ingressgateways) entry.
+
+```shell-session
+$ cat > config.yaml <<EOF
+connectInject:
+  enabled: true
+ingressGateways:
+  enabled: true
+  gateways:
+    - name: ingress-gateway
+      replicas: 2
+      service:
+       type: LoadBalancer
```

Comment:
This will expose their cluster with a public load balancer. Can you add a security note that this is what will happen and that they should ensure they delete the LB afterwards.